sass/components/Dashboard/dropdown.scss

```diff
@@ -0,0 +1,11 @@
+.kubevirt-dropdown__items {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  margin: 0.6em 0;
+
+  .btn-group.btn-group-justified {
```

Comment:
it would be better to pass new argument to `Dropdown` (ie `groupClassName`) if you need to style it

Reply:
ack

Reply:
This is still not addressed.  `btn-group.btn-group-justified` is specific to dropdowns but `DashboardCardActionsBody` may contain anything. When you need to style dropdowns in `DashboardCardActionsBody`  you need to style them directly which will require new Dropdown prop `groupClassName` which will be applied to `ButtonGroup` in `Dropdown.js`